patches/@statechannels+devtools+0.1.4.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/@statechannels/devtools/lib/src/config/env.js b/node_modules/@statechannels/devtools/lib/src/config/env.js
+index b54571f..b8e3bc0 100644
+--- a/node_modules/@statechannels/devtools/lib/src/config/env.js
++++ b/node_modules/@statechannels/devtools/lib/src/config/env.js
+@@ -30,10 +30,16 @@ exports.getEnvBool = getEnvBool;
+ function configureEnvVariables() {
+     const SC_ENV = process.env.SC_ENV;
+     if (SC_ENV) {
+-        const scEnvFile = path.join('../..', '.env.' + SC_ENV);
+-        const scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
++        // node_modules/@statechannels/($simple-hub|$xstate-wallet)
++        let scEnvFile = path.join('../../..', '.env.' + SC_ENV);
++        let scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
+         if (!fs.existsSync(scEnvFileFullPath)) {
+-            throw new Error(`${scEnvFileFullPath} must exist in the monorepo root`);
++            // packages/$app/<xyz>
++            scEnvFile = path.join('../..', '.env.' + SC_ENV);
++            scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
++            if (!fs.existsSync(scEnvFileFullPath)) {
++                throw new Error(`${scEnvFileFullPath} must exist in the monorepo root`);
++            }
```

Comment:
This is a hack that we should probably implement a better fix. Basically this script might run inside `node_modules/@statechannels/<some package>` or it may run inside `packages/<some package/`. cc @kerzhner

Reply:
Do we want something akin to the way `tsconfig.json` etc are searched for?

 https://github.com/sindresorhus/find-up

This can be fixed separately, but we should create an issue for it.

Reply:
An alternative is something like: 
` scEnvFile = path.join(PROJECT_ROOT, '.env.' + SC_ENV);`

Reply:
This makes `PROJECT_ROOT` a parameter of any script which calls `configureEnvVariables`, as opposed to its current usage which is as an expected value inside an `SC_ENV` file which gets used by `start_servers.sh`